src/main/webapp/app/appConfig.ts

```diff
@@ -13,6 +13,7 @@ export type FirebaseProperties = {
   messagingSenderId: string;
   appId: string;
   measurementId: string;
+  isTest?: boolean; // this will be injected by tests and will not come from the backend
```

Comment:
It makes me nervous to see a flag for testing. Is there a way to use mocks to avoid using test code in the production code?